SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py

```diff
@@ -50,6 +52,9 @@
 from Configuration.StandardSequences.Eras import eras
 if eras.fastSim.isChosen():
     hcalSimBlock.hitsProducer = cms.string('famosSimHits')
+if eras.run2_HF_2016.isChosen():
```

Comment:
This should be changed to `toModify()`. Matti is right that `toModify()` is applied immediately. I would like to change that and if I did I would have to preserve the effect that the changes would also be picked up by anything importing that parameter.